source/features/latest-tag-button.tsx

```diff
@@ -118,14 +120,16 @@ async function init(): Promise<false | void> {
 	}
 
 	const defaultBranch = await getDefaultBranch();
+	const compareLink = (
+		<a className="btn btn-sm btn-outline tooltipped tooltipped-ne" href={`/${getRepoURL()}/compare/${latestTag}...${defaultBranch}`}>
+			<DiffIcon/>
+		</a>
+	);
```

Comment:
This is only used inside `if (currentBranch === defaultBranch) {`, so it should be in there